File: baekjoon/1차원 배열/4344.py
C = int(input())
for i in range(C):
  T = list(map(int,input().split()))
  N = T[0]
  scores = T[1:N+1]
  sum = 0
  for j in range(N):
    sum = sum + scores[j]
  avg = sum/N
  result = format(len(list(filter(lambda x: x>avg, scores)))/N * 100, ".3f")
  print(f"{result}%")

File: baekjoon/단계별로 풀어보기/1차원 배열/4344.js
const readline = require('readline')
const rl = readline.createInterface({
  input: process.stdin,
  output: process.stdout,
})

let input = []

rl.on('line', (line) => {
  input.push(line)
}).on('close', () => {
  const C = parseInt(input[0])
  for (i = 1; i <= C; i++) {
    const TC = input[i].split(' ').map((item) => parseInt(item))
    const N = parseInt(TC[0])
    let sum = 0
    for (j = 1; j <= N; j++) {
      sum += TC[j]
    }
    let avg = sum / N
    const mtAvg = (TC.slice(1).filter((item) => item > avg).length / N) * 100
    console.log(`${mtAvg.toFixed(3)}%`)
  }
  process.exit()
})
